ownCloud/Viewer/DisplayHostViewController.swift

```diff
@@ -87,6 +87,17 @@ class DisplayHostViewController: UIViewController {
 		navigationController?.tabBarController?.tabBar.isHidden = false
 	}
 
+	override func prefersHomeIndicatorAutoHidden() -> Bool {
+		return true
+	}
+
+	override func childViewControllerForHomeIndicatorAutoHidden() -> UIViewController? {
```

Comment:
@pablocarmu I don’t understand how this method is related to “home indicator autohidden”? At least not from reading method signature..

Reply:
The home indicator is always managed by the system, we can not hide it if we want.
But you can say to iOS that you prefer for your view controller the home indicator to be hidden.

This calls `prefersHomeIndicatorAutoHidden`. If you return true, basically you're saying to the system that you want it hidden, but is an iOS choice.

If you have child view controllers and you want a different behavior for each one, you need to implement  `childViewControllerForHomeIndicatorAutoHidden`. If you return a view controller that is your child, the system will ask for the child view controller `prefersHomeIndicatorAutoHidden` value instead of the parent one. If you return nil the system asks for the parent `prefersHomeIndicatorAutoHidden`.

`DisplayHostViewController` is the parent view controller of `ImageDisplayViewController`, and we want to hide the home indicator when we're in full screen in the image preview. So we need to override this func and return `ImageDisplayViewController` so it's called its implementation.

I should delete the `prefersHomeIndicatorAutoHidden` implementation in `DisplayHostViewController`.

Reply:
@pablocarmu Ok, got it. Checked Apple docs, was not aware that this API even exists